blueprint/src/exporter/go.mod

```diff
@@ -0,0 +1,38 @@
+module exporter
+
+go 1.17
+
+require (
+	github.com/mypurecloud/platform-client-sdk-go/v65 v65.0.0
+	github.com/prometheus/client_golang v1.12.1
+)
```

Comment:
This should be the same require block

Reply:
That is what I thought too but they were declared separate in all the examples(including those in the documentation) I came across. I couldn't find promhttp in the prometheus package as well.